pandas/tseries/offsets.py

```diff
@@ -2382,7 +2379,7 @@ def apply(self, other):
                     qtr_lens = self.get_weeks(other + self._offset)
 
                 for weeks in qtr_lens:
-                    start += relativedelta(weeks=weeks)
```

Comment:
i would leave the weeks, not real well tested for timedelta; relativedelta may be more intelligent here

Reply:
> i would leave the weeks, not real well tested for timedelta

Not sure I understand.  You mean the stdlib timedelta isn't well-tested?

Reply:
> relativedelta may be more intelligent here

The relativedelta constructor just adds `weeks*7` to the `days` kwarg.